uint32 simulation_step
string agent_name
bool update_job
bool update_remove_job
bool update_request
bool update_reply
bool update_remove
bool update_resources
SOJobMessage[] job
SOJobRemove[] remove_job
SOMeetingReply[] replies
SOMeetingRemove[] remove
SOMeetingRequest[] requests
SOResourceMessage[] resources

